src/pages/api/scrape.js

```
@@ -23,10 +220,29 @@ export async function POST({ request }) {
     const { data } = await axios.get(url);
     const $ = cheerio.load(data);
     
-    // Extract recipe information (this is a basic implementation)
-    // Recipe websites vary widely, so this is a simplistic approach
+    // Try to extract structured data from LD+JSON first
+    const ldJsonData = extractLDJSON($);
+    const structuredRecipe = processLDJSONRecipe(ldJsonData);
     
-    // Try to find the recipe title
+    if (structuredRecipe && structuredRecipe.title && 
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)